src/foam/u2/view/DAOListWithCreateView.js

```diff
@@ -187,7 +191,7 @@ foam.CLASS({
           .end();
       return this.SUPER()
         .addClass(this.myClass('actionBar'))
-        .start(this.VIEW_MORE, { buttonStyle: 'TERTIARY' })
+        .start(this.VIEW_MORE, { buttonStyle: 'TERTIARY' }).show(this.showViewMore)
```

Comment:
Better to just do:
 return this.SUPER()
        .addClass(this.myClass('actionBar'));
 if ( this.showViewMore ) {
  this.start()
     ...
  end();
}

That way you won't add the DOM you don't need.

Reply:
oh, I see. thank you.